src/NoTimeToWaitBundle/Resources/views/Default/nttw6.html.twig

```diff
@@ -0,0 +1,99 @@
+{% extends 'NoTimeToWaitBundle::base.html.twig' %}
+
+{% block title %}No Time to Wait 6: Open (Still) Isn't Enough!{% endblock %}
+
+{% block body %}
+<h1><strong>No Time to Wait!</strong> *</h1>
+
+<h2>The No Time to Wait Conference</h2>
+<p><a href="https://www.beeldengeluid.nl/">Netherlands Institute for Sound and Vision</a> shall host the sixth No Time to Wait conference. No Time to Wait is a FREE three-day conference focused on <em>open media</em>, <em>open standards</em>, and <em>digital audiovisual preservation</em> managed by <a href="{{ path('homepage') }}">MediaArea.net</a>. No Time to Wait has always been a conference that blends in-person and online participation and this year we shall work to emphasize online participation in response to the social, travel, and financial restrictions related to the pandemic.</p>
+<p>We continue last year's theme with "Open Still Isn't Enough" highlighting that simply bringing openness into our work is not enough to propagate its adoption and  accessibility, but that open media and open standards require an environment of skill-sharing, advocacy, collaboration, and community-building to be truly useful.</p>
+<h2>When</h2>
+<p>October 26-28, 2022</p>
+
+<h2>Registration</h2>
+<p>If you'd like to participate in No Time to Wait 6, either in-person or online, please complete <a href="https://bit.ly/nttw6signup">our registration form</a>.</p>
+
+<h2>Call for Proposals</h2>
+<p>Coming soon :D</p>
+
+<h2>What</h2>
+
+<p>The No Time To Wait conference features presentations and discussion on topics such as:</p>
+
+<ul>
+  <li>developing collaborative initiatives to preserve media</li>
+  <li>efforts to extend capabilities and opportunities within archives</li>
+  <li>active open media standardization projects</li>
+  <li>seeking consensus in audiovisual preservation strategy</li>
+  <li>examination of open media use in film and video digitization</li>
+  <li>integration of open source tools into archival workflow</li>
+  <li>examples of cross-community collaboration and skill-sharing</li>
+  <li>developments in open media</li>
+  <li>and more</li>
+</ul>
+
+<p>Follow <a href="https://twitter.com/nttwconf">@nttwconf</a> for news and updates about No Time to Wait. Our recommended hashtag is <a href="https://twitter.com/hashtag/nttw6">#nttw6</a>.</p>
+<p>The conference will be livestreaming on the MediaArea YouTube channel <a href="http://www.youtube.com/c/MediaAreaNetOfficial">available here</a>.</p>
+
+<h2>Where</h2>
+<p><a href="//www.beeldengeluid.nl">Netherlands Institute for Sound and Vision at The Hague</a>, Zeestraat 82, 2518 AD Den Haag, Netherlands.</p>
+
+<p>
+{% image '@NoTimeToWaitBundle/Resources/public/img/page1-1600px-Erfgoed_Gelderland_presentatie_Jesse_de_Vos.pdf.jpg' %}
+  <img src="{{ asset_url }}" alt="exterior image of Netherlands Institute for Sound and Vision" class="img-responsive center-xs" title="Netherlands Institute for Sound and Vision">
```

Comment:
The photo is form Hilversum, it may be misleading, better to have a The Hague photo.